packages/venia-ui/lib/components/ProductOptions/swatch.js

```diff
@@ -40,9 +40,10 @@ const Swatch = props => {
 
     const { handleClick } = talonProps;
 
-    const icon = useMemo(() => {
-        return isSelected ? <Icon src={CheckIcon} /> : null;
-    }, [isSelected]);
+    const checkStyle = useMemo(
+        () => isSelected ? defaultClasses.checkmark : defaultClasses.checkmarkHidden,
```

Comment:
Ordinarily, we want classnames to be semantic, describing the state of an element rather than how we chose to present it. Do we think `checkmark` and `checkmarkHidden` are the state here?

- `selected` & `unselected`?
- `checked` & `unchecked`?
- `on` & `off`?

Reply:
Done, thank you!